dev/IconSource/FontIconSource.cpp

```diff
@@ -18,10 +18,11 @@ winrt::IconElement FontIconSource::CreateIconElementCore()
         fontIcon.Foreground(newForeground);
     }
 
-    if (FontFamily())
+    if (!FontFamily())
     {
-        fontIcon.FontFamily(FontFamily());
+        FontFamily(winrt::FontFamily{ c_fontIconSourceDefaultFontFamily });
```

Comment:
I missed that we were setting this if it was unset. Can we instead grab this into a local variable and set it on the fontIcon only but not modify the FontIconSource?

Reply:
```c++
auto fontFamily = FontFamily();
if (!fontFamily)
{
    fontFamily = winrt::FontFamily{ c_fontIconSourceDefaultFontFamily };
}
fontIcon.FontFamily(fontFamily);
```

Reply:
I can, but is that the right thing to do? I thought it might be better to set it on the FontIconSource so it's not so mysterious how you got a font when none was set. Up to you though. :)

Reply:
I'm ok either way. I prefer to be non-destructive when possible so that's why I was suggesting this path, but you're making the change so I will let you choose. :)